.github/renovate-regex.json

```diff
@@ -0,0 +1,8 @@
+{
+  "jq": {
+    "renovate_datasource": "github-tags",
+    "renovate_depname": "jqlang/jq",
+    "renovate_versioning": "regex:^jq-(?<major>\d+)(\.(?<minor>\d+))?(\.(?<patch>\d+))?$",
```

Comment:
yes, single backslashes.  Failed without backslashes, and fails with double-backslashes.